_data/simple-icons.json

```diff
@@ -9645,6 +9645,12 @@
             "hex": "F74843",
             "source": "https://www.valvesoftware.com/"
         },
+        {
+            "title": "Vans",
+            "hex": "D51A21",
+            "source": "https://www.vans.com.my/about-us",
```

Comment:
the source address is `403 forbidden`, I think it will not be accepted.😆

Reply:
Interesting, the link works fine for me. Can you access any Vans website from where you're based @kerthcet?